mods/d2k/chrome/encyclopedia.yaml

```diff
@@ -49,31 +49,35 @@ Background@ENCYCLOPEDIA_PANEL:
 				Container@ACTOR_INFO:
 					X: PARENT_RIGHT - WIDTH
 					Y: 30
-					Width: PARENT_RIGHT - 190 - 10
+					Width: PARENT_RIGHT - 190
 					Height: PARENT_BOTTOM - 25
 					Children:
-						Background@ACTOR_BG:
-							Width: 150
-							Height: 170
-							Background: dialog3
-							Children:
-								ActorPreview@ACTOR_PREVIEW:
-									X: 1
-									Y: 1
-									Width: PARENT_RIGHT - 2
-									Height: PARENT_BOTTOM - 2
 						ScrollPanel@ACTOR_DESCRIPTION_PANEL:
-							X: 150 + 10
-							Width: PARENT_RIGHT - 150 - 10
-							Height: 170
+							X: 10
+							Width: PARENT_RIGHT - 10
+							Height: PARENT_BOTTOM
 							TopBottomSpacing: 8
 							Children:
-								Label@ACTOR_DESCRIPTION:
+								Label@ACTOR_TITLE:
 									X: 8
 									Y: 8
 									Width: PARENT_RIGHT - 40
+									Height: 25
+									VAlign: Top
+									Align: Center
```

Comment:
Centre alighnment looks really weird when the title is not in the midile

Reply:
it's jarring with longer names

Reply:
I think it looks good as-is. Do you have a specific suggestion on what to change?

Reply:
Ah, I know why the miscommunication. With shorter less than 15 character names it looks like the title has an indentation from the description, and then indentation gets disregarded by longer names

Reply:
I understand what you're saying, but I don't agree. To me, it seems clear that the the title is associated both with the text and the picture, so the exact relationship between the left of the title and the left of the text doesn't matter.

Reply:
Why center the title and not left align it?

Reply:
I agree it's jarring with a centered title because it seems that it should line-up with the paragraph (sometimes it almost does):
![image](https://github.com/OpenRA/OpenRA/assets/1355810/d63c40e9-dde5-4346-8bc2-242becc87514)
![image](https://github.com/OpenRA/OpenRA/assets/1355810/df5c4d2f-8468-4bd8-b7e0-836c9c58b4ee)

I'd suggest to either left-align it with the paragraph or with the image. The paragraph would probably work better as the image has a lot of air around it.